sql/2019/10_SEO/10_04b.sql

```diff
@@ -0,0 +1,14 @@
+#standardSQL
+
+# hreflang implementation values
+
+SELECT
+  hreflang,
+  COUNT(hreflang) AS occurence,
+  ROUND(COUNT(hreflang) * 100 / SUM(COUNT(0)) OVER (), 2) AS occurence_perc
```

Comment:
seems like there are a few inconsistent names for `percentage`, maybe we stick to the full name, or `pct`?

I guess this is also a cross-chapter consistency issue we might run into. the guide has `pct_*` but I didn't follow that in mine 😳